core/src/main/java/nl/nn/adapterframework/filesystem/ExchangeFileSystem.java

```diff
@@ -1039,7 +1044,21 @@ private CredentialFactory getProxyCredentials(){
 
 	private void setMailboxOnService(ExchangeService service, String mailbox){
 		service.getHttpHeaders().put(ANCHOR_HEADER, mailbox);
-//		service.setImpersonatedUserId(new ImpersonatedUserId(ConnectingIdType.SmtpAddress, mailbox));
+		service.setImpersonatedUserId(new ImpersonatedUserId(ConnectingIdType.SmtpAddress, mailbox));
+	}
+
+	private ExchangeFolderReference asFolderReference(String folderName) throws FileSystemException {
+		ExchangeFolderReference reference = new ExchangeFolderReference(folderName, getMailAddress(), null, getMailboxFolderSeparator());
+		reference.setBaseFolderId(getBaseFolderId(reference.getMailbox(), getBaseFolder()));
```

Comment:
De baseFolder geldt alleen voor het static mailaddress, dus dit moet je hier niet doen

Reply:
Zonder base folder zijn acties als MKDIR niet mogelijk, deze zal je altijd nodig hebben op de volgende regel:

`folder.save(reference.getBaseFolderId());`

Reply:
Daarnaast als ik de base folder id niet zet, kan RMDIR ook niet de te verwijderen folder vinden tijdens de call naar findFolder()

Reply:
OK, het kan op verschillende manieren. Het moet wel duidelijk gedocumenteerd worden in de ibisdoc.